fastlane/lanes/build.rb

```diff
@@ -287,6 +288,8 @@ def upload_installable_build(product:)
       message = "#{product} Installable Build: [#{filename}](#{install_url})"
       sh('buildkite-agent', 'annotate', message, '--style', 'info', '--context', "installable-build-#{product}")
     end
+
+    UI.user_error!('🚧 Force crash just to force the CI step to fail so that we can hit retry on it and test the skip_if_exists works as expected')
```

Comment:
To remove once we've validated that retrying the step after it failed (due to this `user_error!`, so after the first upload to S3 succeeded) will lead to the `upload_to_s3` just log the error.

So:
 - On first run, we expect `upload_to_s3` to succeed, log `"Upload Complete"`, and fail on this ad-hoc `user_error!`
 - On second run, we expect `upload_to_s3` to skip the upload, logging `"File already exists in S3 bucket"`, not crash, and still fail but only on this ad-hoc `user_error!` (and not earlier like before)